tests/acceptance/features/bootstrap/TwoFactorTOTPContext.php

```diff
@@ -125,6 +170,62 @@ public function theUserAddsVerificationKeyFromSecretKeyToVerifyUsingWebUI() {
 		}
 	}
 
+	/**
+	 * @When the user re-logs in as :username using the webUI for two-factor authentication
```

Comment:
```suggestion
	 * @When the user re-logs in as :username using the two-factor authentication
```

I think this might be better. What do you think? @HariBhandari07

Reply:
Oh, I got it wrong.
The better wording might be 
`When the user re-logs in as :username to two-factor authentication verification page` ?

Reply:
```
When the user re-logs in as :username to the two-factor authentication verification page
```